stack/eks.py

```diff
@@ -105,10 +105,62 @@
     label="Cluster name",
 ))
 
+custom_eks_ami = Ref(template.add_parameter(
+    Parameter(
+        "CustomEKSAMI",
+        Description="Custom AMI ID for EKS node group",
+        Type="String",
+        Default="",
+    ),
+    group="Elastic Kubernetes Service (EKS)",
+    label="Custom EKS AMI",
+))
+
+use_custom_ami= "UseCustomAMI"
```

Comment:
I think this will fix pre-commit:

```suggestion
use_custom_ami = "UseCustomAMI"
```